docker-compose.yml

```diff
@@ -11,23 +11,29 @@ services:
     environment:
       - HOST=0.0.0.0
       - PORT=8002
-      - APP_URL=https://localhost:8002
+      - APP_URL=http://localhost:8002
```

Comment:
Yeah, I actually have started on this in my fix for #10, because we need SSL for the redirect_uri to work. Something like this?
```
diff --git a/api/dev b/api/dev
index 21576e2..8ea3ae5 100755
--- a/api/dev
+++ b/api/dev
@@ -4,17 +4,17 @@
 set -e
 
 # -- APPLICATION SETTINGS -----------------------------------------------------
 export ENV=${ENV:="localdev"}
 export PROJECT_NAME=${PROJECT_NAME:="tooltool_api"}
 export FLASK_APP=${FLASK_APP:="tooltool_api.flask:app"}
 export APP_SETTINGS=${APP_SETTINGS:=`realpath ./settings.py`}
 export HOST=${HOST:="127.0.0.1"}
-export PORT=${PORT:="8002"}
+export PORT=${PORT:="8010"}
 export DATABASE_URL=${DATABASE_URL:="postgres://tooltooluser:tooltoolpassword@localhost:8102/tooltooldb"}
 #export TASKCLUSTER_ROOT_URL=${TASKCLUSTER_ROOT_URL:="https://stage.taskcluster.nonprod.cloudops.mozgcp.net"}
 export TASKCLUSTER_ROOT_URL=${TASKCLUSTER_ROOT_URL:="https://firefox-ci-tc.services.mozilla.com"}
 
 # -- LOCAL SCRIPT SETTINGS ----------------------------------------------------
 PYENV_DIR=./env
 PYTHON_VERSION=3.7
 
diff --git a/api/docker.d/cert.pem b/api/docker.d/cert.pem
new file mode 100644
diff --git a/api/docker.d/key.pem b/api/docker.d/key.pem
new file mode 100644
diff --git a/api/docker.d/run.sh b/api/docker.d/run.sh
index bc314a5..3028ce0 100755
--- a/api/docker.d/run.sh
+++ b/api/docker.d/run.sh
@@ -1,26 +1,19 @@
-#!/bin/sh
+#!/bin/bash
 set -e
 
 pushd `dirname $0` &>/dev/null
 MY_DIR=$(pwd)
 popd &>/dev/null
 
 EXTRA_ARGS=""
 
 if [ "$ENV" == "localdev" ]
 then
-    CERT="${MY_DIR}/cert.pem"
-    KEY="${MY_DIR}/key.pem"
-
-    # generate certificate if it doesn't exists
-    if [ ! -e "$KEY" ]; then
-      exit 123;
-      rm -f $CERT $KEY
-      openssl req -x509 -newkey rsa:4096 -nodes -out $CERT -keyout $KEY -days 365
-    fi
+    CERT="$MY_DIR/cert.pem"
+    KEY="$MY_DIR/key.pem"
 
     # Local development only - we don't want these in deployed environments
     EXTRA_ARGS="--bind $HOST:$PORT --workers 3 --timeout 3600 --reload --reload-engine=poll --certfile=$CERT --keyfile=$KEY"
 fi
 
 exec gunicorn tooltool_api.flask:app --log-file - $EXTRA_ARGS
diff --git a/docker-compose.yml b/docker-compose.yml
index 7a15336..c3e5b80 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -1,22 +1,22 @@
 version: '3'
 services:
   api:
     build: api
     ports:
-      - "8002:8002"
+      - "8010:8010"
     volumes:
       # This doesn't quite work because the app has to be installed
       # Maybe we should drop the volume mount and rely on rebuilding?
       - ./api:/app
     environment:
       - HOST=0.0.0.0
-      - PORT=8002
-      - APP_URL=http://localhost:8002
+      - PORT=8010
+      - APP_URL=https://localhost:8010
       - APP_SETTINGS=/app/settings.py
       - ENV=localdev
       - DATABASE_URL=postgres://tooltooluser:tooltoolpassword@db/tooltooldb
       # "notasecret"
       - SECRET_KEY=bm90YXNlY3JldA==
       - TASKCLUSTER_SECRET=$TASKCLUSTER_SECRET
       - TASKCLUSTER_CLIENT_ID=$TASKCLUSTER_CLIENT_ID
       - TASKCLUSTER_ACCESS_TOKEN=$TASKCLUSTER_ACCESS_TOKEN
```

Reply:
Pretty sure this is what we want to do so I pushed it. Let me know if it doesn't make sense to you.